kasa/smart/smartdevice.py

```diff
@@ -86,18 +86,24 @@ async def update(self, update_children: bool = True):
                 "get_current_power": None,
             }
 
+        if self._components["device"] >= 2:
```

Comment:
Nice catch! I think we should wait until we get the modules/features branch done and let handling of version checks to the individual modules, or what do you think?

Reply:
I think this version check handling is pretty basic and easy to replace so I think we just get this merged.

Reply:
Fair enough :+1: